src/Workspaces/Core/Portable/Utilities/AsyncLazy`1.cs

```diff
@@ -294,6 +294,13 @@ private Request CreateNewRequest_NoLock()
 
         public override Task<T> GetValueAsync(CancellationToken cancellationToken)
```

Comment:
I'm with the opinion that we should be cancelling here, unless we have data that indicates that it's actually a problem.